packages/components/package.json

```
@@ -0,0 +1,85 @@
+{
+  "name": "@web3-ui/components",
+  "license": "MIT",
+  "version": "1.0.0",
```

can we take advantage of the change in package name to revert these to a pre-release version number?
```suggestion
  "version": "0.0.1",
```

---

Definitely and thanks for doing that. :)